### PR TITLE
Unifies ANS dates

### PR DESCRIPTION
**Pull request by kaw2k**
### Description
ANS had 3 different implementations for times. This converts all uses of time to be milliseconds since the epoch. 

### Test Plan
Updated and passing test cases

### Related Links
https://aptos-org.slack.com/archives/C05NLAKJM9U/p1700245777573069

### CODE REVIEW

File: src/internal/ans.ts [hunk 1]
```diff
@@ -465,7 +470,7 @@ export async function getAccountSubdomains(
     },
   });
 
-  return data.current_aptos_names;
+  return data.current_aptos_names.map(sanitizeANSName);
```
Comment (0xmaayan) [inline src/internal/ans.ts:473]: do we need to validate `current_aptos_names` is not empty?

**Review by 0xmaayan — APPROVED** [review]

File: src/internal/ans.ts (hunk 1 above)
Comment (kaw2k) [reply src/internal/ans.ts:473]: Nope! The indexer always gives back an array, empty or not. `map` here will be a no-op if we get an empty array back. 

image

**Comment by gregnazario** [comment]
@kaw2k please update the CHANGELOG, and we can merge it

**Comment by kaw2k** [comment]
Ok, updated the chnagelog!

**Pull request merged**